src/util/schema-translator.ts

```diff
@@ -0,0 +1,92 @@
+import { IFrontendEngineData } from "@lifesg/web-frontend-engine/components";
+import { EElementType, TElement, TElementMap } from "src/context-providers";
+
+const emailFieldToSchema = (element: TElement) => {
+    const emailFieldSchema = {
+        [element.id]: {
+            label: element.label,
+            uiType: element.type,
+            columns: {
+                desktop: element.columns,
+            },
```

Comment:
even though form builder is supported on desktop only, the generated form needs to be supported on all screens, do add the column size for tablet and mobile